src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -65,8 +65,8 @@ azure-mgmt-privatedns==1.0.0
 azure-mgmt-rdbms==10.2.0b12
 azure-mgmt-recoveryservices==2.5.0
 azure-mgmt-recoveryservicesbackup==7.0.0
+azure-mgmt-redis==14.3.0
 azure-mgmt-redhatopenshift==1.4.0
-azure-mgmt-redis==14.2.0
```

Comment:
All requirements are arranged in alphabetical order， please don't move it up

Reply:
Corrected ordering